ksqldb-parser/src/main/antlr4/io/confluent/ksql/parser/SqlBase.g4

```diff
@@ -297,14 +297,20 @@ primaryExpression
     | MAP '(' (expression ASSIGN expression (',' expression ASSIGN expression)*)? ')'     #mapConstructor
     | STRUCT '(' (identifier ASSIGN expression (',' identifier ASSIGN expression)*)? ')'  #structConstructor
     | identifier '(' ASTERISK ')'                              		                        #functionCall
-    | identifier '(' (expression (',' expression)* (',' lambdaFunction)*)? ')'            #functionCall
+    | identifier'(' expression ',' intervalExpression ')' 						                    #functionCall
+    | identifier '(' (expression (',' expression)*
+                     ((',' lambdaFunction)* | (',' intervalExpression)*))? ')'            #functionCall
```

Comment:
Why are there two different functionCall? Doesn't the 2nd one behave similar to `identifier(expr, interval)`?

Btw, I see other DBs uses an INTERVAL keyword, like `dateadd(time, INTERVAL '5 hours')`. Snowflake, DB2 and Mysql uses the INTERVAL keyword. Should we support it too?